backend/src/govsg/services/govsg-callback.service.ts

```diff
@@ -59,7 +71,10 @@ const parseWebhook = async (
       message: 'Received status webhook from WhatsApp',
       action,
     })
-    await parseTemplateMessageWebhook(body as WhatsAppTemplateMessageWebhook)
+    await parseTemplateMessageWebhook(
```

Comment:
Do we want to remove the log above this on lines 69 - 73? (Sorry, I couldn't tag on the line itself)

Reply:
Good qn. It's fine to have more logs actually... to improve traceability (I learnt this from the fintech team).

The logs could be improved to make troubleshooting even easier, but I might want to defer this cognitive task to a later PR.

Wdyt?

Reply:
Yup, I think its fine to leave cleaning up the logging in a different PR.

wrt having more logs, I think in this case, we might not want the log on lines 69-73 since `parseTemplateMessageWebhook` itself has logging for the different states it encounters (although the logging coverage is not complete)

Perhaps for now we can leave this particular log in. Once we have capacity to clean the logs, then we should remove it since the logging inside `parseTemplateMessageWebhook` is more specific.

Reply:
I see, I'll remove it in the next PR. I don't have opinions on this so I don't mind just getting it done next.